compiler/rustc_mir_dataflow/src/framework/graphviz.rs

```diff
@@ -394,7 +394,7 @@ where
     ) -> io::Result<()> {
         let diffs = StateDiffCollector::run(body, block, self.results.results(), self.style);
 
-        let mut befores = diffs.before.map(|v| v.into_iter());
+        let mut diffs_before = diffs.before.map(|v| v.into_iter());
         let mut afters = diffs.after.into_iter();
```

Comment:
The two variables should be consistently named (I don't care if it's `befores` or `diffs_before`).